<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #f0742f;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: #f0742f;
	padding: 10px;

}


QPushButton::hover
{
	background-color: #fc7c11;
	color: #fff;

}


QPushButton::pressed
{
	background-color: #ff6b35;
	color: #fff;

}
</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:orange;font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;Smart Parking IoT Solution&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>481</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/park1_on.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/park2_on.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/park3_on.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="button_reserve_1">
      <property name="text">
       <string>Reseve 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_reserve_2">
      <property name="text">
       <string>Reserve 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_reserve_3">
      <property name="text">
       <string>Reserve 3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="button_unreserve_1">
      <property name="text">
       <string>unreseve 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_unreserve_2">
      <property name="text">
       <string>unreserve 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_unreserve_3">
      <property name="text">
       <string>unreserve 3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1,0,2,1">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;Temperature C: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="temperature">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">width:50px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Sun :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sun_statut">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>normal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Park Statut :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
